docs/content/standards/closed-loop-token.mdx

```diff
@@ -6,16 +6,31 @@ Using the Closed-Loop Token standard, you can limit the applications that can us
 
 ## Background and use cases
 
-The [Coin standard](./coin.mdx) on Sui is an example of an open-loop system - coins are free-flowing, [wrappable](/concepts/object-ownership/wrapped), [freely transferable](/concepts/dynamic-fields/transfers/custom-rules#the-store-ability-and-transfer-rules) and you can store them in any application. The best real world analogy would be cash - hardly regulated and can be freely used and passed.
+The [Coin standard](./coin.mdx) on Sui is an example of an open-loop system - coins are free-flowing, [wrappable](/concepts/object-ownership/wrapped.mdx), [freely transferable](/concepts/transfers/custom-rules.mdx#the-store-ability-and-transfer-rules) and you can store them in any application. The best real world analogy would be cash - hardly regulated and can be freely used and passed.
 
 Some applications, however, require constraining the scope of the token to a specific purpose. For example, some applications might need a token that you can only use for a specific service, or that an authorized account can only use, or a token that you can block certain accounts from using. A real-world analogy would be a bank account - regulated, bank-controlled, and compliant with certain rules and policies.
 
+## Difference with Coin
+
+```mermaid
```

Comment:
FWIW, I do like the image we had for this a little bit better, but this is OK too (and maybe you can coerce mermaid to produce a more similar output)

Reply:
Yeah, I did too. We should use Mermaid whenever possible, though, to make maintenance easier for all. This was as close as I could get it. If we really don't want to use mermaid, we'll need to use something publicly available with source files.